website/views/options/settings.jade

```diff
@@ -79,7 +79,7 @@ script(type='text/ng-template', id='partials/options.settings.settings.html')
             div.alert.alert-warning(style='padding:2px;margin-top:7px')
               h5=env.t('customDayStart')
               h5=env.t('customDayStartInfo1')
-                a(ng-click='showCustomDayStartInfo = !showCustomDayStartInfo') {{!showCustomDayStartInfo ? env.t('customDayStartInfo2') : env.t('customDayStartInfo3')}}
+                a(ng-click='showCustomDayStartInfo = !showCustomDayStartInfo') {{!showCustomDayStartInfo ? env.t('showMoreMore') : env.t('showMoreLess')}}
               h5(ng-if='showCustomDayStartInfo')!=env.t('customDayStartInfo4')
```

Comment:
I didn't move this "show more/less" link to the bottom of the text area because it's more convenient where it is (you can click it to show more text, which pushes the text box you'll want to use down the page away from your mouse; but then you can click show less without moving your mouse to bring the text box back up).